serializer_impl.hh

```diff
@@ -771,6 +771,19 @@ std::variant<T...> deserialize(Input& in, boost::type<std::variant<T...>> v) {
     return deserialize_std_variant(in, v, idx, std::make_index_sequence<sizeof...(T)>());
 }
 
+template<typename Output, typename T, typename U>
+void serialize(Output& out, const std::pair<T, U>& p) {
+    serialize(out, p.first);
+    serialize(out, p.second);
```

Comment:
I think it's better to use structs rather than pairs. First, pairs are hostile to anyone reading the code. Second (pun unintended), IDL structs can be extended while pairs cannot.

Reply:
I'll drop this commit. I was using pair when I've changed IDL in the bad way, breaking the compatibility. Now it's not used.